lib/api.js

```diff
@@ -129,6 +129,28 @@ class Madge {
 			.filter((dep) => !map[dep]);
 	}
 
+	/**
+	* Return a list of modules that have no dependencies.
+	* @api public
+	* @return {Array}
+	*/
+	leaves() {
+		const tree = this.obj();
```

Comment:
const tree = this.obj();

return Object.keys(tree).filter((key) => !tree[key].length);